src/core/sys/linux/netinet/in_.d

```diff
@@ -13,7 +13,15 @@ import core.sys.posix.config;
 
 public import core.sys.posix.netinet.in_;
 
-version(CRuntime_Glibc)
+version(CRuntime_Glibc)     version = linux_libc;
+version(CRuntime_Musl)      version = linux_libc;
```

Comment:
Make these all `else version`, as we don't support using multiple libc's at once.

Reply:
I noticed this pattern in other parts of druntime. I can change it however here.

Reply:
[I've been cleaning those up where I can](https://github.com/dlang/phobos/pull/6193#discussion_r168917047).